[libdatachannel] links to target "libSRTP::srtp2" but the target was not  found

### Operating system

ubuntu

### Compiler

gcc

### Steps to reproduce the behavior

my vcpkg.json
```json
    {
      "name": "libdatachannel",
      "features": ["srtp"]
    },
```
my CMakeLists.txt
```CMakeLists.txt
if(WIN32)
    target_link_libraries(cfgoclient PRIVATE LibDataChannel::LibDataChannel)
else()
    target_link_libraries(cfgoclient PRIVATE LibDataChannel::LibDataChannelStatic)
endif(WIN32)
```
error
```
CMake Error at /home/ps/workspace/projects/vcpkg/scripts/buildsystems/vcpkg.cmake:639 (_add_library):
[cmake]   Target "cfgoclient" links to target "libSRTP::srtp2" but the target was not
[cmake]   found.  Perhaps a find_package() call is missing for an IMPORTED target, or
[cmake]   an ALIAS target is missing?
[cmake] Call Stack (most recent call first):
[cmake]   CMakeLists.txt:47 (add_library)
```


### Failure logs

```
CMake Error at /home/ps/workspace/projects/vcpkg/scripts/buildsystems/vcpkg.cmake:639 (_add_library):
[cmake]   Target "cfgoclient" links to target "libSRTP::srtp2" but the target was not
[cmake]   found.  Perhaps a find_package() call is missing for an IMPORTED target, or
[cmake]   an ALIAS target is missing?
[cmake] Call Stack (most recent call first):
[cmake]   CMakeLists.txt:47 (add_library)
```

### Additional context

_No response_

--- a/ports/libdatachannel/fix_srtp.patch
+++ b/ports/libdatachannel/fix_srtp.patch
@@ -1,0 +1,13 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index ca0f12e5..8e0e665f 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -326,7 +326,7 @@ else()
+ 	target_compile_definitions(datachannel PUBLIC RTC_ENABLE_MEDIA=1)
+ 	target_compile_definitions(datachannel-static PUBLIC RTC_ENABLE_MEDIA=1)
+ 	if(USE_SYSTEM_SRTP)
+-		find_package(libSRTP REQUIRED)
++		find_package(libSRTP CONFIG REQUIRED)
+ 		if(NOT TARGET libSRTP::srtp2)
+ 			add_library(libSRTP::srtp2 UNKNOWN IMPORTED)
+ 			set_target_properties(libSRTP::srtp2 PROPERTIES

--- a/ports/libdatachannel/portfile.cmake
+++ b/ports/libdatachannel/portfile.cmake
@@ -40,6 +40,10 @@ vcpkg_cmake_install()
 vcpkg_cmake_config_fixup(PACKAGE_NAME LibDataChannel CONFIG_PATH lib/cmake/LibDataChannel)
 vcpkg_fixup_pkgconfig()
 
+if(srtp IN_LIST FEATURES)
+    set(FIND_DEP_SRTP "find_dependency(libSRTP CONFIG)")
+endif()
+
 file(READ "${CURRENT_PACKAGES_DIR}/share/LibDataChannel/LibDataChannelConfig.cmake" DATACHANNEL_CONFIG)
 file(WRITE "${CURRENT_PACKAGES_DIR}/share/LibDataChannel/LibDataChannelConfig.cmake" "
 include(CMakeFindDependencyMacro)
@@ -48,7 +52,9 @@ find_dependency(OpenSSL)
 find_dependency(LibJuice)
 find_dependency(plog CONFIG)
 find_dependency(unofficial-usrsctp CONFIG)
+${FIND_DEP_SRTP}
 ${DATACHANNEL_CONFIG}")
+
 
 file(REMOVE_RECURSE "${CURRENT_PACKAGES_DIR}/debug/include" "${CURRENT_PACKAGES_DIR}/debug/share")
 

--- a/ports/libdatachannel/vcpkg.json
+++ b/ports/libdatachannel/vcpkg.json
@@ -1,6 +1,7 @@
 {
   "name": "libdatachannel",
   "version-semver": "0.20.1",
+  "port-version": 1,
   "description": "libdatachannel is a standalone implementation of WebRTC Data Channels, WebRTC Media Transport, and WebSockets in C++17 with C bindings for POSIX platforms (including GNU/Linux, Android, and Apple macOS) and Microsoft Windows.",
   "homepage": "https://github.com/paullouisageneau/libdatachannel",
   "license": "MPL-2.0",

--- a/versions/baseline.json
+++ b/versions/baseline.json
@@ -4242,7 +4242,7 @@
     },
     "libdatachannel": {
       "baseline": "0.20.1",
-      "port-version": 0
+      "port-version": 1
     },
     "libdatrie": {
       "baseline": "0.2.13",

--- a/versions/l-/libdatachannel.json
+++ b/versions/l-/libdatachannel.json
@@ -1,6 +1,11 @@
 {
   "versions": [
     {
+      "git-tree": "74a0be59e4e852bb220e6e86cfa0dde836f0b449",
+      "version-semver": "0.20.1",
+      "port-version": 1
+    },
+    {
       "git-tree": "b829fa3c4fbd6031bef6399df8747a9b51245cb6",
       "version-semver": "0.20.1",
       "port-version": 0

